### PR TITLE
New example

### PR DESCRIPTION
**Pull request by rockstar**
Update the example code. While doing this, I discovered a couple of issues with the code, and so I updated them, along with the tests.

### CODE REVIEW

**Comment by rockstar** [comment]
This should also fix issues #77 and #24

File: examples/api_test.py [hunk 1]
```diff
@@ -16,24 +16,17 @@
...
-else:
-    print("Whoops!")
-containers = lxd.container_list()
-for x in containers:
-    lxd.container_destroy(x)
+container_config = {
+    'name': CONTAINER_NAME,
+    'source': {'type': 'none'},
+}
+container = lxd.containers.create(container_config, wait=True)
```
Comment (jpic) [inline examples/api_test.py]: Could this example perhaps demonstrate error handling too ?

**Comment by jpic** [comment]
LGTM, it would be great if the example demonstrated error handling too.

**Comment by zulcss** [comment]
+1 from me

**Pull request merged**